### ISSUE TITLE
Windows 10 Update (April 2018) and VVV

### ISSUE THREAD

**Issue by benlumia007** [Varying-Vagrant-Vagrants/VVV#1476]
This isn't an issue, but I would like to point out that if anyone updated Windows 10 to the latest operating system of 1803 version, for some odd reason, it also removes the virtualbox virtual network adapter, so when you try to vagrant up again, it will create a new adapter. 

so just an FYI.

**Comment by tomjn** [comment]
Thanks, does this cause any issues or UI changes a user might encounter?

**Comment by benlumia007** [comment]
at this point, everything else seems fine and it doesn't seem to cause issues with the UI or anything, but I will keep you posted.

**Comment by benjaminlu007** [comment]
@tomjn 

I did a clean install of windows 10 with the new version with version vagrant  2.1.0 and 2.1.1 and virtualbox 5.2.8 and virtualbox 5.2.10

I'm not sure what's going on, but i get this error 

the ssh command responded with a non-zero exit status. Vagrant assumes that this means the command failed. The output for this command should be in the log above. Please read the output to determine what went wrong.

**Comment by benlumia007** [comment]
sorry, used the wrong account,

**Comment by jeanfbrito** [comment]
Same problem here. formating back to 1709 to test.

**Comment by benlumia007** [comment]
@jeanfbrito 
Apparently it was the Windows problem, you may wanna try Vagrant 2.1 or higher with VVV 2.2.1. it has been fixed, i  have tested on 1803 on all of my 5 computers and its working well.

**Comment by jeanfbrito** [comment]
@benjaminlu007 
I just installed all them, from the last versions disponible to download on the sites. Virtualbox 5.2.12 and Vagrant 2.1.1.
Tryed change to VirtualBox 5.1.38 and the problem continues.
I just formatted the PC with the build 1803, then getting back to 1709 will not be a big problem.
When I was using 1709, all worked well, and another computer here just works too, with 1709. I will get back and see what could be.
